GCTC-NTGC/gc-digital-talent
# Admin - Print user table profiles

Resolves #2644. h/t @esizer.

### Bonus

- [updates Loading component padding, background color](https://github.com/GCTC-NTGC/gc-digital-talent/commit/9dc64ec258173407ebc0591f42db8ccaf9f39ed9)


### Steps to Test
1. Refresh containers `docker-compose run --rm maintenance bash refresh_all.sh`
2. Navigate and login as admin user at `/admin`
3. Navigate to Users Table `/en/admin/users`
4. Slect all or a few of the rows using the checkbox on the far right column
6. Scroll to table footer at the bottom
7. Near **Selected actions** click **Print** button
8. See print preview screen with rows of users selected
